### DISCUSSION TITLE
LED su Mini PC Acemagic T9Plus

### DISCUSSION THREAD

**Discussion by DjMayone** [indomus/forum#3165] [HUB: Home Assistant]
Ciao a tutti,
ho installato Home Assistant OS su un MiniPC Acemagic T9Plus (quello con i led sul case che funzionano in modalità "arcobaleno).
Volevo sapere se è possibile controllare i led tramite HAOS. Sarebbe interessante poterli controllare in modo da poter configurare anche delle notifiche "visive" sul MiniPc stesso. Ho trovato uno script a questo link https://www.rigacci.org/wiki/doku.php/doc/appunti/hardware/t9plus_mini_pc_rgb_led_control ma non saprei come integrarlo in HAOS.
Avete qualche suggerimento?
Grazie mille!

**Comment by KC-inDomus** [comment]
prova così https://indomus.it/componenti/componente-home-assistant-python-scripts/

**Reply by DjMayone** [reply]
Grazie mille! Sono riuscito ad importare lo script però quando lo vado ad eseguire dai "Servizi" mi viene restituito l'errore: "Invocazione del servizio python_script.rgb non riuscita. Unknown error".
Qualche idea?
Grazie ancora.

**Comment by DjMayone** [comment]
Ciao a tutti, torno su questo post.
Ho testato lo script indicato da riga di comando in bash e funziona correttamente. Ora mi piacerebbe in qualche modo importarlo in interfaccia grafica. Qualcuno saprebbe darmi indicazioni su come utilizzare lo script in HAOS?
Grazie come sempre!
